### PR TITLE
[CI] Update dependencies on travis flaky build

### PR DESCRIPTION
**Pull request by amogkam**
## Why are these changes needed?
* Fixes issue introduced by #15812

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by fishbone — APPROVED** [review]

**Review by ijrsvt — APPROVED** [review]
LGTM Thanks!

**Comment by richardliaw** [comment]
nice work here!

**Comment by ericl** [comment]
Is this actually ok? (e.g., if I remove `TF_VERSION=2.1.0 TFP_VERSION=0.8 TORCH_VERSION=1.6` will the right versions of TF and torch still be tested for RLlib)?

cc @sven1977

File: .travis.yml [hunk 1]
```diff
@@ -92,7 +92,7 @@ matrix:
         - RAY_DEFAULT_BUILD=1
         - RAY_CYTHON_EXAMPLES=1
         - RAY_USE_RANDOM_PORTS=1
-        - RLLIB_TESTING=1 TF_VERSION=2.1.0 TFP_VERSION=0.8 TORCH_VERSION=1.6
+        - RLLIB_TESTING=1
```
Comment (ericl) [inline .travis.yml:95]: Why this change?

**Comment by amogkam** [comment]
@ericl we moved the dependency pinning to rllib specific requirement files in their PR https://github.com/ray-project/ray/pull/15812, rather than doing it in the CI.
